content/en/boilerplates/auto-san-validation.md

```diff
@@ -0,0 +1,3 @@
+---
+---
+Istio has `auto_sni` and `auto_san_validation` enabled by default. This means, whenever there is no explicit `sni` set in your `DestinationRule`, transport socket SNI for new upstream connections will be set based on the downstream HTTP host/authority header. And if there are no `subjectAltNames` set in the `DestinationRule` when `sni` is unset, `auto_san_validation` will kick in, and the upstream presented certificate for new upstream connections will be automatically validated based on the downstream HTTP host/authority header.
```

Comment:
```suggestion
Istio has `auto_sni` and `auto_san_validation` enabled by default. This means, whenever there is no explicit `sni` set in your `DestinationRule`, transport socket SNI for new upstream connections will be set based on the downstream HTTP host/authority header. If there are no `subjectAltNames` set in the `DestinationRule` when `sni` is unset, `auto_san_validation` will kick in, and the upstream-presented certificate for new upstream connections will be automatically validated based on the downstream HTTP host/authority header.
```

Do you want to move the {{< warning >}} into the boilerplate?

Reply:
> Do you want to move the {{< warning >}} into the boilerplate?

I changed the warning to tip based on an offline review comment from @leosarra , have moved to the boilerplate but let us see if lint complains